packages/webapp/src/containers/Finances/CustomRevenueType/EditCustomRevenue.jsx

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import PureSimpleCustomType from '../../../components/Forms/SimpleCustomType';
+import { HookFormPersistProvider } from '../../hooks/useHookFormPersist/HookFormPersistProvider';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { updateCustomRevenueType } from '../saga';
+import { revenueTypeByIdSelector, revenueTypesSelector } from '../../revenueTypeSlice';
```

Comment:
I went with the other naming convention for that one -- `revenueTypeSelector`. It seemed to be used more but I personally prefer the more explicit version (`ById`) so please feel free to rename it. However as is, this will crash on import.

Reply:
It looks like I forgot to review my imports for edit/readonly, I swore I checked to match it up haha I even forgot the persist form in there.